_pages/en_US/installing-boot9strap-(kartdlphax).txt

```diff
@@ -70,9 +74,28 @@ On the **target 3DS** (the 3DS that you are trying to modify):
 1. Select "Start" on the **source 3DS** once it has detected the **target 3DS**
 1. Once multiplayer has loaded, navigate to `Grand Prix` -> `50cc` -> (any driver) -> `Mushroom Cup` -> `OK`
 1. Wait a while (a percentage should be displayed on the **source 3DS**)
-1. If the exploit was successful, the **target 3DS** will have booted into SafeB9SInstaller
+1. If the exploit was successful, the **target 3DS** will have booted into the 3DS ROP xPloit Injector
+1. Press (X) to use unSAFE_MODE
+  + The target 3DS will automatically shut down
+  + You won't need the source 3DS to complete any further steps on this guide
 
-#### Section IV - Installing boot9strap
+#### Section V - unSAFE_MODE
+
+If your (Right/Left Shoulder), (D-Pad Up), or (A) buttons do not work, you will need to follow [Seedminer into Bannerbomb3](seedminer). For assistance with this matter, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask, in English, for help.
```

Comment:
If we can menuhax67 people then it's probably better to direct people to NH instead, as Seedminer doesn't directly link to BB3 and people may just stumble onto the TWN instructions that won't work for them.
(At least until we have a menuhax67 guide up whether here or on wiki. I dunno, there's probably a better way to work this out logistically.)

Reply:
Also, might be better to switch to the new style taken from #2086